packages/entities/entities-plugins/src/components/free-form/shared/BooleanField.vue

```diff
@@ -9,7 +9,9 @@
   <KCheckbox
     v-else
     v-bind="fieldAttrs"
+    :id="field.path.value"
```

Comment:
For per instance `id`s, I think we should use `useId` from `vue` directly.

Reply:
Why `useId` is better than a meaningful id?